opendebates/views.py

```diff
@@ -182,6 +182,13 @@ def vote(request, id):
             }
     voter, created = Voter.objects.get_or_create(email=form.cleaned_data['email'])
 
+    if voter.user and voter.user != request.user:
+        # anonymous user can't use the email of a registered user
+        msg = 'That email is registered. Please login and try again.'
+        return HttpResponse(
```

Comment:
It looks like this view can be called via Ajax or not (see line 173 above) so we need to return the errors in different ways, depending.